src/overlays/actors/ovl_Bg_Iknin_Susceil/z_bg_iknin_susceil.c

```diff
@@ -41,7 +41,7 @@ ActorInit Bg_Iknin_Susceil_InitVars = {
     (ActorFunc)BgIkninSusceil_Draw,
 };
 
-static s32 unused = 0;
+static s32 sPad = 0;
```

Comment:
Its `sBssPad` in the other files.

Reply:
This isn't bss though. It's just regular data padding.

Reply:
Ah yeah. I forgot IDO doesn't put zero initialized in BSS.